Formula/podman.rb

```diff
@@ -4,6 +4,7 @@ class Podman < Formula
   url "https://github.com/containers/podman/archive/v3.2.2.tar.gz"
   sha256 "70f70327be96d873c83c741c004806c0014ea41039e716545c789b4393184e79"
   license "Apache-2.0"
```

Comment:
```suggestion
  license "Apache-2.0"
  revision 1
```
Since there's a change of dependencies we should bump the `revision` here.